File: 프로그래머스/unrated/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int add = 0;
        int mul = 1;
        for(int i = 0; i <num_list.length; i++){
            add += num_list[i];
        }
         for(int i = 0; i <num_list.length; i++){
            mul *= num_list[i];
        }
        if(mul < add*add ){
            answer = 1;
        }
        else{
            answer = 0;
        }
        return answer;
    }
}

File: 프로그래머스/unrated/181929. 원소들의 곱과 합/원소들의 곱과 합.js
function solution(num_list) {
    let total = 0
    let multiply = 1
    let length = num_list.length
    for(let i = 0; i < length; i++){
        total += num_list[i]
        multiply *= num_list[i]
    }
    if(multiply < total*total){
        return 1
    }
    else {
        return 0
    }
}